app/code/Magento/Sales/Test/Mftf/Test/AdminCreateOrderToVerifyApplyAndRemoveCouponCodeTest.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateOrderToVerifyApplyAndRemoveCouponCodeTest">
+        <annotations>
+            <stories value="Create Order with offline payment methods"/>
+            <title value="Create Order to verify apply and remove coupon code test"/>
+            <description value="Create Order to verify apply and remove coupon code test"/>
+            <severity value="CRITICAL"/>
+            <testCaseId value="MC-37386"/>
+            <group value="sales"/>
+        </annotations>
+        <before>
+            <!-- Login as admin -->
```

Comment:
Please remove the comments between the steps, they are pretty much the same content already on the stepKey attribute, I don't think it brings any benefit.